src/scripts/main.js

```diff
@@ -1,7 +1,27 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const message = document.createElement('div');
+  const titleMessage = document.createElement('h2');
+  const textMessage = document.createElement('p');
+
+  message.classList.add(`notification`);
+  message.classList.add(type);
```

Comment:
You can chain the 'add' method to add multiple classes at once, like so: 'element.classList.add(class1, class2, ...)'.